packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -533,6 +545,37 @@ export class GrpcPlugin extends BasePlugin {
     }
   }
 
+  /**
+   * Returns a TagMap on a Metadata object if it exists and is well-formed, or
+   * null otherwise.
+   * @param metadata The Metadata object from which TagMap should be retrieved.
+   */
+  static getTagContext(metadata: grpcTypes.Metadata): TagMap|null {
+    const metadataValue = metadata.getMap()[GRPC_TAGS_KEY] as Buffer;
+    // Entry doesn't exist.
+    if (!metadataValue) return null;
+    try {
+      const tags = deserializeBinary(metadataValue);
+      // Value is malformed.
+      if (!tags) return null;
+      return tags;
+    } catch (ignore) {
+    }
+    return null;
```

Comment:
Nit: would it make sense for the `return null;` to be above within the `catch` block?

Reply:
Sure, done